src/app/containers/RelatedTopics/index.jsx

```diff
@@ -47,13 +45,10 @@ RelatedTopicsWrapper.propTypes = {
 
 const RelatedTopics = ({ topics }) => {
   const { service, script, translations, dir } = useContext(ServiceContext);
-  const { variant, pageType } = useContext(RequestContext);
-  const { enabled: cpsTopicTagsAreEnabled } = useToggle('cpsTopicsTags');
-  const { enabled: optimoTopicTagsAreEnabled } = useToggle('optimoTopicsTags');
+  const { variant } = useContext(RequestContext);
   const isEnabled =
-    pageType === ARTICLE_PAGE
-      ? optimoTopicTagsAreEnabled
-      : cpsTopicTagsAreEnabled;
+    process.env.SIMORGH_APP_ENV === 'test' ||
```

Comment:
```suggestion
    process?.env?.SIMORGH_APP_ENV === 'test' ||
```
You may want to use [optional chaining here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)  to prevent this failing when the `RelatedTopics` component is rendered clientside; this would happen if you navigated from one optimo article to another I think. The optional chaining would just short circuit to undefined which would make the topic tags not appear which is a safe option before you take this out for go live.

Reply:
Good shout, thanks @andrewscfc